lib/modules/versioning/conan/common.ts

```diff
@@ -30,7 +30,12 @@ export function makeVersion(
 export function cleanVersion(version: string): string {
   if (version) {
     return version
-      .replace(regEx(/,|\[|\]|"|include_prerelease=|loose=|True|False/g), '')
+      .replace(
```

Comment:
please do these changes to versioning in a separate PR

Reply:
Done -> https://github.com/renovatebot/renovate/pull/26588